stac_fastapi/extensions/stac_fastapi/extensions/core/transaction.py

```diff
@@ -34,7 +33,7 @@ class PutItem(ItemUri):
 class PutCollection(CollectionUri):
     """Update Collection."""
 
-    collection: stac.Collection = attr.ib(default=Body(None))
+    collection: Collection = attr.ib(default=Body(None))
```

Comment:
@jonhealy1 I'm not sure why but when we use this class (attr) we don't get the validation/serialization 😬 

```python
from stac_fastapi.extensions.core.transaction import PutCollection

>>> PutCollection(collection_id='yo', collection={})
PutCollection(collection_id='yo', collection={})

>>> print(type(PutCollection(collection_id='yo', collection={}).collection))
<class 'dict'>
```

you may have seen this also in the elastic search backend